share/rocm/cmake/ROCMInstallTargets.cmake

```diff
@@ -289,6 +289,7 @@ function(rocm_export_targets)
     foreach(INCLUDE ${PARSE_INCLUDE})
         rocm_install(FILES ${INCLUDE} DESTINATION ${CONFIG_PACKAGE_INSTALL_DIR})
         get_filename_component(INCLUDE_BASE ${INCLUDE} NAME)
+        list(APPEND PARSE_INCLUDE_NAME ${INCLUDE_BASE})
```

Comment:
Whatever variable we use should be cleared at the start of the function, to avoid accidentally using values from the calling scope.